docker/opensource.md

```diff
@@ -86,6 +86,23 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # composer install
 # phpunit
 ```
+## capella
 
+* [capella](https://github.com/AlexSpradling/Capella)
+* I have added `CONTRIBUTING.rst`
+
+```
+$ git clone https://github.com/AlexSpradling/Capella.git
+$ cd capella
+$ docker run -it --name capella-dev -w /opt -v %cd%:/opt python:3.11 bash
```

Comment:
On linux this would be

docker run -it --name capella-dev -w /opt -v $(pwd):/opt python:3.11 bash